scripts/fix_files.sh

```diff
@@ -1,12 +1,12 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 FIX_FILES_VERSION="1.5"
 
 echo "Fix-Files version $FIX_FILES_VERSION"
 echo ""
 
 # Check bash version
-if [[ ${BASH_VERSINFO[0]} < 5 && (${BASH_VERSINFO[0]} < 4 || ${BASH_VERSINFO[1]} < 1) ]]; then
+  if [[ ${BASH_VERSINFO[0]} < 5 && (${BASH_VERSINFO[0]} == 4 && ${BASH_VERSINFO[1]} < 1) ]]; then
```

Comment:
Why would you change this condition. It breaks the sanity check and only prevent bash 4.0 to continue (all 3.x, 2.x, 1.x version will be able to continue without being catched). The previous test was working fine, didn't it?

Reply:
Yesterday my solution looked good to me. Now I clearly see weird logic in it =)